src/commands/command-manager.ts

```diff
@@ -0,0 +1,21 @@
+import * as vscode from 'vscode';
+import { multiInject, injectable, inject } from 'inversify';
+import TYPES from '../types';
+import { ICommand } from './icommand';
+import { AllCommand } from './all-command';
+
+ @injectable()
+export class CommandManager {
+  constructor(
+    @multiInject(TYPES.Command) private commands: ICommand[],
+    @inject(TYPES.AllCommand) private allCommand: AllCommand
```

Comment:
I'm guessing you actually meant to use the interface here, instead of the concrete class?
```suggestion
    @inject(TYPES.AllCommand) private allCommand: IAllCommand
```